js/scripts.js

```diff
@@ -1,7 +1,7 @@
 
-var buttons = document.getElementsByClassName('btn'){
+var buttons = document.getElementsByClassName('btn');
+
+for (var i=0; i<buttons.length; i++) { 
```

Comment:
1) add spaces :)
2) why your alert is empty?
3) move `buttons.length` to new variable and use it here
4) remember that buttons is an array (use your iterator to get each element from this array)